bin/node/Cargo.toml

```diff
@@ -16,56 +16,56 @@ targets = ["x86_64-unknown-linux-gnu"]
 name = "aleph-node"
 
 [dependencies]
-structopt = "0.3.8"
+clap = { version = "3.0", features = ["derive"] }
 log = "0.4"
-serde = "1"
-serde_json = "1"
+serde = "1.0"
+serde_json = "1.0"
 futures = "0.3"
-hex-literal = "0.3.1"
-libp2p = "0.39.1"
-hex = "0.4.2"
+hex = "0.4"
+hex-literal = "0.3"
+libp2p = "0.43"
```

Comment:
Is there a particular reason for bumping it above the version that substrate depends on? I was working on polkadot 9.19 update and I noticed this introduces a conflicting dependency version:
```
error: failed to select a version for `zeroize`.
    ... required by package `chacha20poly1305 v0.9.0`
    ... which satisfies dependency `chacha20poly1305 = "^0.9"` of package `snow v0.9.0`
    ... which satisfies dependency `snow = "^0.9.0"` of package `libp2p-noise v0.35.0`
    ... which satisfies dependency `libp2p-noise = "^0.35.0"` of package `libp2p v0.43.0`
    ... which satisfies dependency `libp2p = "^0.43"` of package `aleph-node v0.4.0 (/home/hansu/aleph/aleph-node/bin/node)`
versions that meet the requirements `>=1, <1.5` are: 1.4.3, 1.4.2, 1.4.1, 1.4.0, 1.3.0, 1.2.0, 1.1.1, 1.1.0, 1.0.0

all possible versions conflict with previously selected packages.

  previously selected package `zeroize v1.5.4`
    ... which satisfies dependency `zeroize = "^1.5.4"` of package `sc-network v0.10.0-dev (https://github.com/paritytech/substrate.git?branch=polkadot-v0.9.19#174735ea)`
    ... which satisfies git dependency `sc-network` (locked to 0.10.0-dev) of package `aleph-node v0.4.0 (/home/hansu/aleph/aleph-node/bin/node)`

failed to select a version for `zeroize` which could resolve this conflict
```

Reply:
Update: I worked a bit more on that and this is cargo.lock hell :/ Let's not have two versions of lib-p2p living together in the same crate, pleeeaaaassse ;)